src/api/delivery-needed/index.js

```diff
@@ -12,67 +77,25 @@ exports.deliveryNeededRequestHandler = async (req, res) => {
       .send({ message: `Error fetching requests: ${requestErr}` });
   }
 
-  const {
-    code,
-    crossStreetFirst,
-    crossStreetSecond,
-    meta,
-    neighborhoodAreaSeeMap,
-    firstName
-  } = fields;
+  const regularRequestsPromises = requestObj
+    .filter(r => !r.fields[forDrivingClusters])
+    .map(makeFeature);
 
-  const requestsWithCoordsPromises = requestObj.map(async r => {
-    let metaJSON = {};
-    let slackPermalink = {};
-    const location = await fetchCoordFromCrossStreets(
-      `
-      ${r.fields[crossStreetFirst]},
-      ${r.fields[crossStreetSecond]},
-      Brooklyn
-      `
-    );
+  const clusterRequestsPromises = requestObj
+    .filter(r => Boolean(r.fields[forDrivingClusters]))
+    .map(makeFeature);
 
-    try {
-      metaJSON = JSON.parse(r.fields[meta]);
-    } catch {
-      console.error(
-        "[deliveryNeededRequestHandler] could not parse meta",
-        r.fields.meta
-      );
-    }
+  const regularRequests = await Promise.all(regularRequestsPromises);
+  const clusterRequests = await Promise.all(clusterRequestsPromises);
 
-    try {
-      const channel = metaJSON.slack_channel;
-      const timestamp = metaJSON.slack_ts;
-      slackPermalink = await slackapi.chat.getPermalink({
-        channel,
-        message_ts: timestamp
-      });
-    } catch {
-      console.error(`[deliveryNeededRequestHandler] could not fetch slack URL
-        for requestCode: ${r.fields[code]} channel: ${metaJSON.slack_channel} and timestamp: ${metaJSON.slack_ts}`);
+  return res.send({
+    requests: {
```

Comment:
send requests as separate FeatureCollections. this means the whole payload is no longer valid geoJSON but these child elements are. could not find way to include multiple FeatureCollections in a geoJSON.